pkgs/tools/misc/autosuspend/default.nix

```diff
@@ -0,0 +1,87 @@
+{ lib
+, stdenv
+, pkgs
+, python3
+, fetchFromGitHub
+}:
+
+let
+  sphinx-autodoc-typehints = with python3.pkgs; buildPythonPackage rec {
+    pname = "sphinx-autodoc-typehints";
+    version = "1.12.0";
+    disabled = pythonOlder "3.6";
+
+    buildInputs = [ setuptools-scm setuptools ];
+    propagatedBuildInputs = [ sphinx ];
+
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "0llzia8clzq7vvpwf0ifr1cxjk73fjg3d79rn60p5170vgcifdhr";
+    };
+  };
+  sphinx-issues = with python3.pkgs; buildPythonPackage rec {
+    pname = "sphinx-issues";
+    version = "1.2.0";
+    disabled = pythonOlder "3.6";
+
+    buildInputs = [ setuptools-scm setuptools ];
+    propagatedBuildInputs = [ sphinx ];
+
+    src = fetchPypi {
+      inherit pname version;
+      sha256 = "1wq9yp66xq9scazzc05vgh1n84cyf0gl64vgf2fc1i3sdirr8ll4";
+    };
+  };
+in
+  with python3.pkgs; buildPythonApplication rec {
+    pname = "autosuspend";
+    version = "3.1.3";
+
+    src = fetchFromGitHub {
+      owner = "languitar";
+      repo = "autosuspend";
+      rev = "6adc8f812ebcaeae83d2b0725ca37d7b433b7f4a";
+      sha256 = "0lms0q5nmywlxph3qqh7vp13pcj49l1fh0yxl23ljfnl5v9lwhjl";
+    };
+
+    propagatedBuildInputs = [
+      dbus-python
+      mpd2
+      requests
+      lxml
+      icalendar
+      dateutil
+      tzlocal
+      freezegun
+      jsonpath-ng
+      portalocker
+      psutil
+    ];
+
+    buildInputs = [
+      pytest
+      pytest-mock
+      pytest-httpserver
+      python-dbusmock
+      sphinx
+      sphinx-issues
+      sphinx-autodoc-typehints
+      sphinxcontrib_plantuml
```

Comment:
```suggestion
      sphinxcontrib-plantuml
```

Reply:
I had to edit the entry in `pkgs/top-level/python-packages.nix` and change the underscore in a hyphen there in order to make it work. Running a quick `git grep` shows that no other package in nixpkgs depends on `sphinxcontrib-plantuml`, but still I renamed a package and I would like to make sure it's alright

Reply:
It was added with #132421